app/__init__.py

```diff
@@ -30,5 +30,10 @@ def create_app(test_config=None):
     migrate.init_app(app, db)
 
     # Register Blueprints here
+    from .task_routes import tasks_bp
+    app.register_blueprint(tasks_bp)
+
+    from .goal_routes import goals_bp
+    app.register_blueprint(goals_bp)
```

Comment:
Imports can be combined to one line: 

```suggestion
    from .task_routes import tasks_bp, goals_bp
    app.register_blueprint(tasks_bp)
    app.register_blueprint(goals_bp)
```